Wireframe/index.css

```diff
@@ -0,0 +1,52 @@
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+  } 
+
+ header {
+   background-color: #F1F1F1;
+   text-align: center;
+   padding: 20px;
+  } 
+ 
+ #logo {
+    display: block;
+    margin: 0 auto;
+ }
+ button{
+  padding: 10px, 10px;
+  background-color: #F1F1F1;
+  border-radius: 5px;
+
+ }
+
+  article:first-child {
```

Comment:
if you made some _general_ rules for article, and then set _just_ the rules that were _different_ for first child, it might make it clearer how CSS cascades work. Give it a try!